configure.ac

```diff
@@ -12,7 +12,6 @@ AC_PROG_LIBTOOL
 AC_HEADER_STDC
 AM_PATH_PYTHON3
 AX_PYTHON3_DEVEL
-AX_CODE_COVERAGE
```

Comment:
Why is this changed in this PR?

Reply:
Good catch, I was trying to make the build work locally and it snuck into my PR, I'll revert this change and update

Reply:
please revert